src/components/form/form.tsx

```diff
@@ -0,0 +1,113 @@
+import { Button, Field, FormGeneric, FormPanel } from '@mdb/flora';
+import {
+  jobFunctions,
+  institutionTypes,
+  teachingStatuses,
+} from 'src/lib/registration';
+import FormProps from './types';
+import styles from './styles';
+
+export default function Form({
+  isOpen,
+  onClose,
+}: FormProps): JSX.Element | null {
+  return isOpen ? (
+    <div data-testid="form-modal" sx={styles.FormModalWrapper}>
+      <div sx={styles.FormModal}>
+        {/* @ts-ignore */}
+        <FormPanel
+          title="Join the MongoDB Educator Community"
+          onClose={onClose}
+        >
+          <FormGeneric
+            inverse
+            customStyles={{
+              width: ['100%', '440px', null, '100%'],
+            }}
+          >
+            <Field
```

Comment:
Nitpick, but would it be better to use a map to loop over each `Field` component? So something like so:

```
const fields = [{ name: "lastname", label: "Last Name", component: "text-input" },{ name: "email", label: "email", component: "text-input" }];

fields.map(({ name, label, component }) => (
	<Field component={component} name={name} label={label} />
))
```

Reply:
Had a similar thought, was going to wait for integration to see what the data looked like but will just include that in this branch.